### DISCUSSION TITLE
How to use Tmxlite on windows?

### DISCUSSION THREAD

**Discussion by XiaJack** [fallahn/tmxlite#170] [General]
I always have some problems, Can you launch a tutorial？
![屏幕截图 2024-09-08 223721](https://github.com/user-attachments/assets/e6c9c5e0-c56c-491b-b73d-15001a229637)

**Comment by eXpl0it3r** [comment]
You may just want to use the [`FindTMXLITE.cmake` module](https://github.com/fallahn/tmxlite/blob/master/tmxlite/cmake/modules/FindTMXLITE.cmake), so you can use the normal `find_package` command.

If you specify include/link directories, you should always use the `target_*` specific types and not do it "globally".

Are you sure tmxlite is located at `D:/msys64/minwg64/include`?
